README.md

```diff
@@ -1,41 +1,130 @@
+# La Boulangerie
+
+Created by: 
+- Backend 
+  - [Eric Belongea](https://www.linkedin.com/in/eric-belongea/) | [GitHub](https://github.com/EricBelongea)
+  - [Cory Powell](https://www.linkedin.com/in/coryrpow/) | [GitHub](https://github.com/coryrpow)
+  - [Clay O'Leary](https://www.linkedin.com/in/john-clay-oleary/) | [GitHub](https://github.com/Captainlearyo)
+- Frontend
+  - [Bobby Steckline](https://www.linkedin.com/in/rsteckline/) | [GitHub](https://github.com/rsteckline)
+  - [Deanna Stevens](https://www.linkedin.com/in/deanna-sofia-stevens/) | [GitHub](https://github.com/dsstevens)
+
+## Directory
+- [Getting Started]()
```

Comment:
Not sure if you've done this link thing before, but I can show you how to get it once this is merged in to main.